raster-foundry/raster-foundry
# [Front End] Cleans up front-end map controls

## Overview
- Cleaned up all map controls throughout the app to be consistent.
- Added tooltips to add map controls on hover
- Updated font icon to include missing icons

### Checklist

### Demo
<img width="330" alt="screen shot 2018-08-29 at 2 20 51 pm" src="https://user-images.githubusercontent.com/1928955/44807398-79ff8380-ab97-11e8-9c59-6d17f29d7828.png">
<img width="295" alt="screen shot 2018-08-29 at 2 21 17 pm" src="https://user-images.githubusercontent.com/1928955/44807399-79ff8380-ab97-11e8-9f11-967414b26fa2.png">
<img width="879" alt="screen shot 2018-08-29 at 2 22 42 pm" src="https://user-images.githubusercontent.com/1928955/44807400-79ff8380-ab97-11e8-9e4f-6d57e13b5607.png">
<img width="298" alt="screen shot 2018-08-29 at 2 22 51 pm" src="https://user-images.githubusercontent.com/1928955/44807401-79ff8380-ab97-11e8-9d29-9f13717d58c4.png">
<img width="456" alt="screen shot 2018-08-29 at 2 22 57 pm" src="https://user-images.githubusercontent.com/1928955/44807402-79ff8380-ab97-11e8-91b8-f222511d66d7.png">

## Testing Instructions

 * View map controls in project, lab, scene modal, public project page.
 * Make sure each action has a tooltip
 * Ensure actions still do what is expected of them (especially lab exports and measure tool)

Closes #XXX
